etl/steps/data/garden/unesco/2024-06-25/education_sdgs.countries.json

```diff
@@ -0,0 +1,262 @@
+  "MKD": "North Macedonia",
+  "MLI": "Mali",
+  "MLT": "Malta",
+  "MMR": "Myanmar",
+  "MNE": "Montenegro",
+  "MNG": "Mongolia",
+  "MNP": "Northern Mariana Islands",
+  "MOZ": "Mozambique",
+  "MRT": "Mauritania",
+  "MSR": "Montserrat",
+  "MTQ": "Martinique",
+  "MUS": "Mauritius",
+  "MWI": "Malawi",
+  "MYS": "Malaysia",
+  "MYT": "Mayotte",
+  "NAM": "Namibia",
+  "NCL": "New Caledonia",
+  "NER": "Niger",
+  "NFK": "Norfolk Island",
+  "NGA": "Nigeria",
+  "NIC": "Nicaragua",
+  "NIU": "Niue",
+  "NLD": "Netherlands",
+  "NOR": "Norway",
+  "NPL": "Nepal",
+  "NRU": "Nauru",
+  "NZL": "New Zealand",
+  "OMN": "Oman",
+  "PAK": "Pakistan",
+  "PAN": "Panama",
+  "PCN": "Pitcairn",
+  "PER": "Peru",
+  "PHL": "Philippines",
+  "PLW": "Palau",
+  "PNG": "Papua New Guinea",
+  "POL": "Poland",
+  "PRI": "Puerto Rico",
+  "PRK": "North Korea",
+  "PRT": "Portugal",
+  "PRY": "Paraguay",
+  "PSE": "Palestine",
+  "PYF": "French Polynesia",
+  "QAT": "Qatar",
+  "REU": "Reunion",
+  "ROU": "Romania",
+  "RUS": "Russia",
+  "RWA": "Rwanda",
+  "SAU": "Saudi Arabia",
+  "SDN": "Sudan",
+  "SEN": "Senegal",
+  "SGP": "Singapore",
+  "SHN": "Saint Helena",
+  "SJM": "Svalbard and Jan Mayen",
+  "SLB": "Solomon Islands",
+  "SLE": "Sierra Leone",
+  "SLV": "El Salvador",
+  "SMR": "San Marino",
+  "SOM": "Somalia",
+  "SPM": "Saint Pierre and Miquelon",
+  "SRB": "Serbia",
+  "SSD": "South Sudan",
+  "STP": "Sao Tome and Principe",
+  "SUR": "Suriname",
+  "SVK": "Slovakia",
+  "SVN": "Slovenia",
+  "SWE": "Sweden",
+  "SWZ": "Eswatini",
+  "SXM": "Sint Maarten (Dutch part)",
+  "SYC": "Seychelles",
+  "SYR": "Syria",
+  "TCA": "Turks and Caicos Islands",
+  "TCD": "Chad",
+  "TGO": "Togo",
+  "THA": "Thailand",
+  "TJK": "Tajikistan",
+  "TKL": "Tokelau",
+  "TKM": "Turkmenistan",
+  "TLS": "East Timor",
+  "TON": "Tonga",
+  "TTO": "Trinidad and Tobago",
+  "TUN": "Tunisia",
+  "TUR": "Turkey",
+  "TUV": "Tuvalu",
+  "TZA": "Tanzania",
+  "UGA": "Uganda",
+  "UKR": "Ukraine",
+  "URY": "Uruguay",
+  "USA": "United States",
+  "UZB": "Uzbekistan",
+  "VAT": "Vatican",
+  "VCT": "Saint Vincent and the Grenadines",
+  "VEN": "Venezuela",
+  "VGB": "British Virgin Islands",
+  "VIR": "United States Virgin Islands",
+  "VNM": "Vietnam",
+  "VUT": "Vanuatu",
+  "WLF": "Wallis and Futuna",
+  "WSM": "Samoa",
+  "YEM": "Yemen",
+  "ZAF": "South Africa",
+  "ZMB": "Zambia",
+  "ZWE": "Zimbabwe",
+  "XDN": "Sudan (former)",
+  "ZZA": "Channel Islands",
+  "WB: European Union": "European Union (27)",
+  "WB: East Asia & Pacific": "East Asia and Pacific (WB)",
+  "WB: Europe & Central Asia": "Europe and Central Asia (WB)",
+  "WB: Latin America & Caribbean": "Latin America & Caribbean (WB)",
+  "WB: Middle East & North Africa": "Middle East & North Africa (WB)",
+  "WB: North America": "North America (WB)",
+  "WB: South Asia": "South Asia (WB)",
+  "WB: Sub-Saharan Africa": "Sub-Saharan Africa (WB)",
+  "WB: High income (July 2023)": "High income",
```

Comment:
We tend to have these as 'High income countries', not a huge deal but it is just a little clearer what is being shown.